### ISSUE TITLE
Model performance is not same as reference implementation

### ISSUE THREAD

**Issue by DrChainsaw** [DrChainsaw/neuralODE4j#10]
The performance of both the resnet reference model (which does not use any of the added stuff from this repo) and the odenet model is much worse than the performance from the reference repo.

After completing 200 epochs with the resnet model, the validation accuracy was still only about 92%. 

As a comparison, the same model from the reference repo reaches accuracy of about 98% after the first epoch (vs about 88% in this repo). This holds true even after replacing layers which are not implemented in dl4j with ones which are (e.g replace group norm with batch norm).

**Comment by DrChainsaw** [comment]
Performance of ResNet is now very similar to what is reported in paper. After #11 and #13 the test error is ~0.72% vs reported 0.41% in paper and this might be within error bounds. 

OdeNet has not been fully evaluated yet given it runs much slower. Short experiments point towards roughly the same error as the resnet though.

Next step is to add the same data augmentation as in the pytorch repo to see if this closes the gap for the reference.

**Comment by DrChainsaw** [comment]
After #15 the error when using the res downsampling method is in range 0.6-0.4% which is near enough for me.
